src/css/index.css

```diff
@@ -10,7 +11,11 @@ html, body {
   color: white;
   font-family: 'Roboto Mono', monospace;
 }
-
+@media screen and (max-width: 560px) and (min-width: 320px) {
+ html{
+   font-size: 15px !important;
```

Comment:
It is generally not recommended to use `!important` still I don't know much css but that's what I've heard. Any views to this @singhpratyush